src/core/modes/registrations/mapMode.tsx

```diff
@@ -12,8 +12,5 @@ export function registerMapMode(modesControlAtom: ModesControlsAtom) {
     onClick() {
       currentModeAtom.setCurrentMode.dispatch('map');
     },
-    onChange(isActive) {
```

Comment:
it looks like we need to remove this code section in all mode files to be consistent 
src/core/modes/registrations

Reply:
agree, removed